src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -12,7 +12,8 @@
   "runtime": {
     "rest": {
       "enabled": false,
-      "path": "/api"
+      "path": "/api",
+      "request-body-strict": true
```

Comment:
was this field auto-added?

Reply:
CLI generates the config file with the `request-body-strict` field even for cosmos.

To update the reference config files with the new roles I'm introducing, I used the config generator scripts (after the addition of necessary DAB commands) to generate these files.